lib/ansible/plugins/terminal/asa.py

```diff
@@ -44,25 +43,26 @@ def on_open_shell(self):
             self.disable_pager()
 
     def disable_pager(self):
-        cmd = {u'command': u'no terminal pager'}
         try:
-            self._exec_cli_command(u'no terminal pager')
+            self.cli('no terminal pager')
         except AnsibleConnectionFailure:
             raise AnsibleConnectionFailure('unable to disable terminal pager')
 
     def on_become(self, passwd=None):
         if self._get_prompt().strip().endswith(b'#'):
```

Comment:
Since the return of _get_prompt() is in format I think b'' prefix is not required.